### PR TITLE
Story/vspc 1

### PR DESCRIPTION
**Pull request by Neehaarika**
VSPC-1
When adding a space link, when a user clicks on the image the icon should move to the new location

### CODE REVIEW

File: vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp [hunk 1]
```diff
@@ -29,8 +30,9 @@ $( document ).ready(function() {
 	
 	$("#addSpaceLinkButton").click(function(e) {
 		$("#bgImage").on("click", function(e){
-		    e.preventDefault();
-		    var icon = $('<span data-feather="navigation-2" class="flex"></span>');
+			e.preventDefault();
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp]: indentation?

File: vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp [hunk 2]
```diff
@@ -56,6 +58,7 @@ $( document ).ready(function() {
 	$("#cancelSpaceLinkBtn").click(function() {
 		storeX = null;
 		storeY = null;
+		$("#arrow").removeAttr("id");
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp]: if the create procedure is cancelled, the arrow should disappear.

File: vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp [hunk 3]
```diff
@@ -29,8 +30,9 @@ $( document ).ready(function() {
 	
 	$("#addSpaceLinkButton").click(function(e) {
 		$("#bgImage").on("click", function(e){
-		    e.preventDefault();
-		    var icon = $('<span data-feather="navigation-2" class="flex"></span>');
+			e.preventDefault();			
+		    $("#arrow").remove();
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp]: what if there is no element with that id? does it just ignore it or does it throw an error?

File: vspace/pom.xml [hunk 4]
```diff
@@ -22,7 +22,7 @@
 		<db.url>jdbc:mysql://localhost:3306/vspace</db.url>
 		<db.user>vspace</db.user>
 		<db.password>vspace</db.password>
-		<uploaded.files.path>/Users/jdamerow/Software/VirtualSpaces2.0/uploads/</uploaded.files.path>
+		<uploaded.files.path>/Users/neehaarikadasari/Documents/</uploaded.files.path>
```
[the remark this answers is not in the document]
Comment (Neehaarika) [reply vspace/pom.xml]: Okay

File: vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp (hunk 1 above)
Comment (Neehaarika) [reply vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp]: Corrected the indentation in my second Task.

File: vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp (hunk 2 above)
Comment (Neehaarika) [reply vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp]: Sure. Will remove the arrow upon Cancelling.

File: vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp (hunk 3 above)
Comment (Neehaarika) [reply vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp]: No, it does not throw an error it is just ignoring that case.

File: vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp (hunk 1 above)
Comment (Neehaarika) [reply vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp]: I Fixed this as part of the second task

File: vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp [hunk 5]
```diff
@@ -21,14 +21,15 @@ $( document ).ready(function() {
...
+
 	}
 	</c:forEach>
 	
 	var storeX;
 	var storeY;
 	
 	$("#addSpaceLinkButton").click(function(e) {
-		$("#bgImage").on("click", function(e){
+		$("#bgImage").one("click", function(e){
```
[the remark this answers is not in the document]
Comment (Neehaarika) [reply vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp]: this does not. I changed this code and resubmitted the correct version of it I believe.

**Pull request closed without merging**